lib/active_storage/attached/many.rb

```diff
@@ -1,27 +1,41 @@
+# Representation of multiple attachments to a model.
 class ActiveStorage::Attached::Many < ActiveStorage::Attached
   delegate_missing_to :attachments
 
+  # Returns all the associated attachment records.
   def attachments
     @attachments ||= ActiveStorage::Attachment.where(record_gid: record.to_gid.to_s, name: name)
   end
 
+  # Associates one or several attachments with the current record, saving
+  # them to the database.
   def attach(*attachables)
     @attachments = attachments | Array(attachables).flatten.collect do |attachable|
       ActiveStorage::Attachment.create!(record_gid: record.to_gid.to_s, name: name, blob: create_blob_from(attachable))
     end
   end
 
+  # Checks the presence of attachments.
+  #
+  #   class Gallery < ActiveRecord::Base
+  #     has_many_attached :photos
+  #   end
+  #
+  #   Gallery.new.photos.attached? # => false
   def attached?
     attachments.any?
   end
 
+  # Directly purges (i.e. destroys) each associated attachment.
```

Comment:
Should explain that purge here means = destroy the blob, destroy attachment, delete the file on the service.